server/views/pages/incident.njk

```diff
@@ -0,0 +1,108 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% set pageTitle = applicationName + " - Incident" %}
+{% set mainClasses = "app-container govuk-body" %}
```

Comment:
not a big deal for throw-away pages, but `app-container` is not a valid class

Reply:
👌 [Removed non-existent CSS class](https://github.com/ministryofjustice/hmpps-incident-reporting/pull/131/commits/6df6f1e61d51fee35df3c19d7ac48989c5dd289e)